Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rlua-builders"
-version = "0.1.3"
+version = "0.1.4"
```

Comment:
I *think* we should bump the version for the derive crate as well. I'll do that.

Reply:
Oh yeah, good point